### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by mgorski-mg**

### CODE REVIEW

**Comment by mgorski-mg** [comment]
Guys, you can check the error while building the app.
It's connected with PWA, but I'm not sure how to fix it. Could you help?

**Review by AntonRaichuk — CHANGES_REQUESTED** [review]
Don't you want to replace S3 with S2 everywhere?

**Comment by mgorski-mg** [comment]
> Don't you want to replace S3 with S2 everywhere?

haha true xD I did it at night and I thought about S3 Bucket xD. Sure will do

File: .github/workflows/deploy-dev.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+on:
+  push:
+    branches:
+      - main
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment: dev
+    env:
+      S3_URL: ${{ secrets.S3_URL }}
```
Comment (AntonRaichuk) [inline .github/workflows/deploy-dev.yaml:11]: S3? :)

Comment (mgorski-mg) [reply .github/workflows/deploy-dev.yaml:11]: It's S3 Bucket Url
we have API Url and S3 Bucket Url

Comment (mgorski-mg) [reply .github/workflows/deploy-dev.yaml:11]: It is used in this build
`run: yarn s3p sync -y -t $S3_URL`

**Review by AntonRaichuk — APPROVED** [review]

**Pull request merged**